perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json

```diff
@@ -18,7 +18,7 @@
       "config_file_flags_as_json": {
         "file-cache": {
           "max-size-in-mb": -1,
-          "download-file-for-random-read": true
+          "cache-file-for-range-read": true
```

Comment:
@Tulsishah please check if this change will work. If not please suggest the correct way.

Reply:
This will work.